setup/artifacts/github.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: single
+title:  "Configuring GitHub Webhooks"
+sidebar:
+  nav: setup
+---
+
+{% include toc %}
+
+Spinnaker can be configured to listen to changes to a repository in
+[GitHub](https://github.com). These steps will take you through how to
+configure a GitHub artifact account to be able to download
```

Comment:
"These steps show how to configure a GitHub artifact account so that Spinnaker can download artifacts."